homework/english/pikus_denis/lecture1/Homework.md

```diff
@@ -0,0 +1,46 @@
+# Упражнения Present Simple и Present Continuous
+## Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+### 1. Does your mum work today?
```

Comment:
I believe that in this case it is not a permanent action and should be used Present Continuous